src/components/avatars/OcAvatar.vue

```diff
@@ -48,18 +64,126 @@ export default {
       required: false,
       default: "",
     },
-
+    /**
+     * The size of the avatar in pixels
+     */
     width: {
       type: Number,
       required: false,
       default: 50,
     },
+    /**
+     * The initials get computed by the component itself
+     */
+    initials: {
```

Comment:
Why do we need the initials as a prop? From my understanding we can have the initials of the userName as a computed property. IMO the parser is not needed for us.

Reply:
Successfully re-re-rebased this and implemented the feedback, ready for re-review